### PR TITLE
Improved the "Naming" section

### PR DESCRIPTION
**Pull request by nakov**
Improved the "Naming" section in the Solidity docs. Added naming examples. Improved consistency.

### CODE REVIEW

File: docs/style-guide.rst [hunk 1]
```diff
@@ -696,49 +696,51 @@ indistinguishable from the numerals one and zero.
...
 Function Names
 ==============
 
-Functions should use mixedCase.
+Functions should use mixedCase. Examples: ``getBalance``, ``transfer``, ``verifyOwner``, ``addMember``, ``changeOwner``.
 
 
-Function Arguments
-==================
+Function Arguments Names
```
Comment (chriseth) [inline docs/style-guide.rst]: I think it should be `Function Argument Names` (only one `s`).

File: docs/style-guide.rst [hunk 2]
```diff
@@ -696,49 +696,51 @@ indistinguishable from the numerals one and zero.
...
+
+Function arguments should use mixedCase. Examples: ``initialSupply``, ``account``, ``recipientAddress``, ``senderAddress``, ``newOwner``.
 
 When writing library functions that operate on a custom struct, the struct
 should be the first argument and should always be named ``self``.
 
 
-Local and State Variables
-=========================
+Local and State Variables Names
```
Comment (chriseth) [inline docs/style-guide.rst]: Please also remove the `s` here.

**Review by chriseth — CHANGES_REQUESTED** [review]
Please remove the surplus `s` characters and then this is ready to go. Thanks!

File: docs/style-guide.rst (hunk 1 above)
Comment (nakov) [reply docs/style-guide.rst]: I agree

File: docs/style-guide.rst (hunk 2 above)
Comment (nakov) [reply docs/style-guide.rst]: Thanks.

**Review by chriseth — APPROVED** [review]

**Pull request merged**